src/biokbase/narrative/narrativeio.py

```diff
@@ -32,6 +33,9 @@
 
 obj_ref_regex = re.compile('^(?P<wsid>\d+)\/(?P<objid>\d+)(\/(?P<ver>\d+))?$')
 
+MAX_METADATA_STRING_LEN = 900
```

Comment:
FYI, the limit is for bytes, not characters. If you have strings with non-ASCII characters you could pass the check below but exceed the workspace limit.